doc/cla/individual/stef090.md

```diff
@@ -0,0 +1,9 @@
+Croatia, 2017-10-31
+
```

Comment:
no need, if you are signing the corporate CLA, no need to sign the individual one